solution-jivanov/users.js

```diff
@@ -0,0 +1,132 @@
+
+// Require needed libraries and modules
+const Service = require('./services/users.js');
+
+module.exports.create = async (event, context) => {
+    console.log("users/create context= ", context);
+    console.log("users/create event= ", event);
+
+    let httpCode = 200; // default is success

```

Comment:
What would happen if an error occurs before the `try` block?

Reply:
if an error occurs before the "try" block then it will be thrown and that service (AWS Lambda) will end with 500 error. That would be immediately caught by our monitoring service (and/or log ingestion one), we will be alerted and can investigate right away (and/or roll back the latest deployment). We should be alerted for all 500s for any of our services.
Of course, we should do our best to avoid such a situation and be very careful of what we put before the "try" block, if anything at all. We prefer to always catch all errors and display a polite/user-friendly message in case of an error.
In this particular case there's nothing that can really throw (due to the setup event is always an object; not because those statements don't throw in general), but that's a good point you make, let me try to minimize that risk further, just pushed an update about that. 

Also, we should probably have a "soft" deployments strategy in place, in order to reduce the impact of issues in general. Or at least blue-green deployments. A proper deployment strategy would help us reduce the risk of issues, downtime and overall impact. As well as recover faster.